cmd/otelcontribcol/builder-config.yaml

```
@@ -6,216 +6,216 @@ dist:
   otelcol_version: 0.93.0
```

```suggestion
  otelcol_version: 0.94.0
```
I think the script likely needs to be updated, or we could just remove this setting.

---

doh! that one's on me.  no script, but I did `:%s/v0.93.0/v0.94.0/g` and should not have had `v` in there...